apps/web-giddh/src/app/inventory/inventory.component.ts

```diff
@@ -120,6 +122,15 @@ export class InventoryComponent implements OnInit, OnDestroy, AfterViewInit {
             this.isMobileScreen = result.matches;
         });
 
+        this.breakPointObservar.observe([
```

Comment:
Avoid creating redundant subscriptions. This subscription is already present in constructor, club them as discussed